examples/align/rs-align.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include <librealsense2/rs.hpp>
 #include "example-imgui.hpp"
+#include "../examples/example-utils.hpp"
```

Comment:
refactor so that `examples-utils.hpp` is listed in `examples.hpp` so the change will be transparent to the users of the latter
Drop relative path and use local notation "examples-utils.hpp"